scripts/lib/libvirt/libvirt_setup.py

```diff
@@ -219,8 +219,11 @@ def compute_config(args, cpu_flags=cpuflags(), machine=None):
         target_dev=targetdevprefix + 'a',
         bootorder=args.bootorder)
 
-    return get_config(merge_dicts(values, configopts),
-                      os.path.join(TEMPLATE_DIR, "compute-node.xml"))
+    config = get_config(merge_dicts(values, configopts),
+                        os.path.join(TEMPLATE_DIR, "compute-node.xml"))
+    if not hypervisor_has_virtio(libvirt_type):
+        config = re.sub(r"<address type='pci'[^>\n]*>", "", config)
```

Comment:
+1 for the fix

but I would replace the line in `<address type='pci' slot='0x0a'/>`  with `$pci_address`. This will clarify also if the fix really needs to replace the one in 'network` or not, instead of delegating in the documentation of `re.sub()` to see if replaces every instance or only one.